src/server/index.tsx

```diff
@@ -27,16 +27,15 @@ const [hostname, port] = process.env["LEMMY_UI_HOST"]
 const extraThemesFolder =
   process.env["LEMMY_UI_EXTRA_THEMES_FOLDER"] || "./extra_themes";
 
-// Commenting out for now, since this broke iOS / webkit browsers.
-// if (!process.env["LEMMY_UI_DEBUG"]) {
-//   server.use(function (_req, res, next) {
-//     res.setHeader(
-//       "Content-Security-Policy",
-//       `default-src 'none'; connect-src 'self' ${wsUriBase}; img-src * data:; script-src 'self' 'unsafe-inline' 'unsafe-eval'; style-src 'self' 'unsafe-inline'; form-action 'self'; base-uri 'self'`
-//     );
-//     next();
-//   });
-// }
+if (!process.env["LEMMY_UI_DEBUG"]) {
+  server.use(function (_req, res, next) {
+    res.setHeader(
+      "Content-Security-Policy",
+      `default-src 'none'; connect-src 'self' ${wsUriBase}; img-src * data:; script-src 'self' 'unsafe-inline' 'unsafe-eval'; style-src 'self' 'unsafe-inline'; form-action 'self'; base-uri 'self'`
+    );
+    next();
+  });
+}
```

Comment:
So this means CSP only runs in prod, not in debug right? Like i said before, i dont like this because we might break things during dev, and not notice until its deployed. Why not simply set a different `connect-src` depending on the debug env var?

Reply:
Correct. Eruda can't work with CSP, its remote javascript, so I need a flag to be able to disable it during dev.

Reply:
What's sad too is I at least found a good repo that replicates rust Results and Options, but it doesn't deserialize correctly :(